wd_core.au3

```diff
@@ -76,8 +76,11 @@ Global Const $_WD_LOCATOR_ByLinkText = "link text"
 Global Const $_WD_LOCATOR_ByPartialLinkText = "partial link text"
 Global Const $_WD_LOCATOR_ByTagName = "tag name"
 
+Global Const $_WD_JSON_RawResponse = ""

```

Comment:
I don't see why this is needed.

Reply:
This is the same kind of need as in case `$_WD_ERROR_Success = 0` and `$_WD_DEBUG_None = 0`
Just for not using Magic Numbers

Reply:
IMO, an empty string isn't the same thing as a magic number. This doesn't add any value; it just adds obscurity. So I can't see approving this addition.

Reply:
So why this following variables are delcared?
```
Global $_WD_DRIVER = "" ; Path to web driver executable
Global $_WD_DRIVER_PARAMS = "" ; Parameters to pass to web driver executable
Global $_WD_SESSION_DETAILS = "" ; Response from _WD_CreateSession

```

Reply:
> This doesn't add any value; it just adds obscurity.

Why `obscurity`
I propose such code:
```
	If $vSubNode = Default Then $vSubNode = $_WD_JSON_RawResponse

	If IsBool($vSubNode) Then $vSubNode = ($vSubNode) ? $_WD_JSON_Value : $_WD_JSON_RawResponse

	If IsString($vSubNode) Then

```

without $_WD_JSON_RawResponse it should look like following snippet:
```
	If $vSubNode = Default Then $vSubNode = ""

	If IsBool($vSubNode) Then $vSubNode = ($vSubNode) ? $_WD_JSON_Value : ""

	If IsString($vSubNode) Then
```
Now reading the code my understanding of the code is smaller.

So maybe like this:
```
	If $vSubNode = Default Then $vSubNode = "" ; Return raw JSON response from web driver
	If IsBool($vSubNode) Then $vSubNode = ($vSubNode) ? $_WD_JSON_Value : "" ; Return JSON value or raw JSON response from web driver

	If IsString($vSubNode) Then
```

Then the code will not contain this constant, but it will be easier to know how it works

Reply:
Yes, that's better. But remember that the header already defines that "" = returns the entire response in JSON format from web driver. Also, I would prefer this --
```
	If IsBool($vSubNode) Then $vSubNode = ($vSubNode) ? $_WD_JSON_Value : "" ; Return JSON value node if $vSubNode = True
```

Reply:
so take a look at: `$_WD_JSON_RawResponse removal` commit